modules/indexing/src/test/java/org/apache/ignite/testsuites/IgniteBinaryCacheQueryTestSuite4.java

```diff
@@ -101,6 +102,8 @@
 
     ScanQueryTransactionsUnsupportedModesTest.class,
     ScanQueryTransactionIsolationTest.class,
+
+    ScanQueryUpdateTtlTest.class,
```

Comment:
It's strange that we have all scan-query tests in indexing-module suite. Looks like they are not related to indexing and should be in core-module. But ok, let's keep it as is.